uint8 packet_type
---
bool result
